sample-status-updater-infrastructure/pom.xml

```
@@ -14,9 +14,9 @@
         <dependency>
             <groupId>life.qbic</groupId>
             <artifactId>sample-status-updater-domain</artifactId>
-            <version>1.3.0-SNAPSHOT</version>
+            <version>1.2.4</version>
```

```suggestion
            <version>1.3.0-SNAPSHOT</version>
```
We need to stay with the current version